app/views/agent/AgentClientIdView.scala.html

```diff
@@ -49,7 +49,7 @@ <h1 class="govuk-heading-l">@messages("agentClientId.heading")</h1>
         )
 
         @govukButton(
-            ButtonViewModel(messages("site.continue")).withAttribute("id", "submit")
+            ButtonViewModel(messages("site.continue")).withAttribute("id", "submit"): Button
```

Comment:
I'm not sure about this. We don't normally put return types in the view like this. I think what it actually wanted was to format it differently: `.withAttribute("id" -> "submit")`